saleor/plugins/webhook/tests/subscription_webhooks/test_create_deliveries_for_taxes.py

```diff
@@ -471,7 +484,306 @@ def test_order_calculate_taxes(
+        "__typename": "CalculateTaxes",
+        "taxBase": {
+            "address": {"id": to_global_id_or_none(order.shipping_address)},
+            "currency": "USD",
+            "discounts": [],
+            "channel": {"id": to_global_id_or_none(order.channel)},
+            "lines": [
+                {
+                    "chargeTaxes": True,
+                    "productName": order_line.product_name,
+                    "productSku": "SKU_A",
+                    "quantity": order_line.quantity,
+                    "sourceLine": {
+                        "__typename": "OrderLine",
+                        "id": to_global_id_or_none(order_line),
+                    },
+                    "totalPrice": {"amount": float(expected_total_price_amount)},
+                    "unitPrice": {
+                        "amount": float(
+                            expected_total_price_amount / order_line.quantity
+                        )
+                    },
+                    "variantName": order_line.variant_name,
+                }
+            ],
+            "pricesEnteredWithTax": True,
+            "shippingPrice": {"amount": float(expected_shipping_price.amount)},
+            "sourceObject": {
+                "__typename": "Order",
+                "id": to_global_id_or_none(order),
+            },
+        },
+    }
+
+
+@freeze_time("2020-03-18 12:00:00")
+def test_order_calculate_taxes_entire_order_voucher(
+    draft_order_with_voucher, webhook_app, permission_handle_taxes, shipping_zone
+):
+    # given
+    order = draft_order_with_voucher
+    expected_shipping_price = Money("2.00", order.currency)
+
+    voucher = draft_order_with_voucher.voucher
+    voucher.type = VoucherType.ENTIRE_ORDER
+    voucher.save(update_fields=["type"])
+
+    discount_amount = Decimal("10")
+    order_discount = order.discounts.first()
+    order_discount.value = discount_amount
+    order_discount.save(update_fields=["value"])
+
+    order.base_shipping_price = expected_shipping_price
+    order.shipping_price = TaxedMoney(
+        net=expected_shipping_price, gross=expected_shipping_price
+    )
+    order.save(
+        update_fields=[
+            "base_shipping_price_amount",
+            "shipping_price_net_amount",
+            "shipping_price_gross_amount",
+        ]
+    )
+
+    manager = get_plugins_manager(allow_replica=False)
+    fetch_order_prices_if_expired(order, manager, order.lines.all(), True)
+
+    shipping_method = shipping_zone.shipping_methods.first()
+    order.shipping_method = shipping_method
+    webhook_app.permissions.add(permission_handle_taxes)
+    webhook = Webhook.objects.create(
+        name="Webhook",
+        app=webhook_app,
+        target_url="http://www.example.com/any",
+        subscription_query=TAXES_SUBSCRIPTION_QUERY,
+    )
+    event_type = WebhookEventSyncType.ORDER_CALCULATE_TAXES
+    webhook.events.create(event_type=event_type)
```

Comment:
don't we want to use fixtures here? like `subscription_calculate_taxes_for_order`?